About.html

```diff
@@ -0,0 +1,184 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+  <title>About</title>
+</head>
+<body>
+  <header class="top bg-dark">
+    <ul>
+      <li><a href="https://www.facebook.com/" aria-label="Facebook" target="_blank"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-facebook" viewBox="0 0 16 16">
+        <path d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"/>
+      </svg></a></li>
+      <li><a href="https://twitter.com/home?lang=en" aria-label="Twitter" target="_blank"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-twitter" viewBox="0 0 16 16">
+        <path d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"/>
+      </svg></a></li>
+      <li><a href="/">English</a></li>
+      <li><a href="/">My Page</a></li>
+      <li><a href="/">Logout</a></li>
+    </ul>
+  </header>
+
+  <nav class="navbar navbar-expand-md navbar-light bg-light px-5">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="./index.html"> <img class="img-fluid" id="log" src="./assets/images/java1.png" alt="logo"> JavaScript </a>
+      <button class="navbar-toggler"
+       type="button"
+       data-bs-toggle="collapse"
+        data-bs-target="#navmenu" aria-controls="navmenu" 
+        aria-expanded="false" 
+        aria-label="Toggle navigation">
+
+        <span class="navbar-toggler-icon"></span> 
+      </button>   
+      <div class="collapse navbar-collapse justify-content-end" id="navmenu">
+        <ul class="navbar-nav  ms-auto">
+          <li class="nav-item">
+            <a class="nav-link"id="activ" aria-current="page" href="./index.html">Home</a>
+          </li>
+          <li class="nav-item">
+         
+            <a class="nav-link" href="#events">Events</a>
+          
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#sponsor-list">Sponsors</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="imp" href="#sponsor-list"><button class="btn">US Community</button></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
- [ ] @MICHAELKITH kindly check on your font weight as previously suggested currently its too light and not as bold as in the design check the comparison below:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/217654825-29e12178-249d-474f-b98a-781d1b191b13.png) | ![image](https://user-images.githubusercontent.com/61408860/217654913-64ecd18d-13c4-4535-9060-46881d35f8f7.png)